tg.js

```diff
@@ -77,10 +77,10 @@ purchase.error May naganap na error
 purchase.error.click dito</a> subukan muli!
 purchase.error.limit Maaaring lumagpas ka sa iyong pang-araw-araw na limitasyon sa transaksyon. 
 settings.reset Mag-reset
-settings.reset.confirm Sigurado ka bang mag-reset ng inyong settings?
+settings.reset.confirm Sigurado ka ba na i-reset ang inyong settings?
 settings.import iangkat
 settings.export iluwas
```

Comment:
```suggestion
settings.export iexport
```

Not used in proper context, pag-angkat and pag-luwas is for trade/economic contexts